### ISSUE TITLE
IC-9700 ptt failure

### ISSUE THREAD

**Issue by mdblack98** [Hamlib/Hamlib#571]
IC-9700 getting
Command rejected by the rig while setting PTT on

**Comment by mdblack98** [comment]
The get_ptt immediately follow the set_ptt.  So it appears some rigs may not be able to get_ptt soon after set_ptt.
We will allow get_ptt to fail and retry

**Comment by mdblack98** [comment]
RC2 installation went fine on W10.  Made a Q65-30A QSO OK on 6m using a TS-590SG.  Switched configuration to the Icom 9700.  Tried FT8 and it was OK.  Tried Q65-30B on 144.170 and after one CQ it turned off the enable transmit.  I turned enable transmit back on and it did two CQs, turned the enable transmit off, and produced the following rig error:

Hamlib error: Command rejected by the rig
icom.c(3572):icom_set_ptt return
rig.c(2549):rig_set_ptt return while setting PTT off
